backend/backEnd_constants.py

```diff
@@ -14,29 +15,36 @@ def __init__(self, cavityNum, rackObject):
         self.statusPV = PV(self.pvPrefix + STATUS_SUFFIX)
         self.severityPV = PV(self.pvPrefix + SEVERITY_SUFFIX)
 
-        self.faultPVs = []
-        for fault in faults:
-            # Decided to use timeout of 0.01 seconds after some trial and error
-            timeout = 0.01
-            if fault.level == "RACK":
-                if fault.rack != self.rack.rackName:
+        self.faults: List[Fault] = []
+
+        for csvFault in CSV_FAULTS:
+
+            if csvFault["Level"] == "RACK":
+
+                # Rack A cavities don't care about faults for Rack B and vice versa
+                if csvFault["Rack"] != self.rack.rackName:
                     continue
-                prefix = self.rack.pvPrefix
-            elif fault.level == "CAV":
-                prefix = self.pvPrefix
-            else:
-                prefix = self.cryomodule.pvPrefix
 
-            self.faultPVs.append((fault, PV(prefix + fault.suffix,
-                                            connection_timeout=timeout)))
+            prefix = csvFault["PV Prefix"].format(LINAC=self.linac.name,
+                                                  CRYOMODULE=self.cryomodule.name,
+                                                  RACK=self.rack.rackName,
+                                                  CAVITY=self.number)
+
+            self.faults.append(Fault(tlc=csvFault["Severity"],
+                                     severity=csvFault["Severity"],
```

Comment:
Should `tlc=csvFault["Severity"]` be `tlc=csvFault["Three Letter Code"]`?

Reply:
OH MY GOD A BUG THANK YOU